Modules/posixmodule.c

```diff
@@ -12179,23 +12179,8 @@ os_cpu_count_impl(PyObject *module)
 {
     int ncpu = 0;
 #ifdef MS_WINDOWS
-    /* Vista is supported and the GetMaximumProcessorCount API is Win7+
-       Need to fallback to Vista behavior if this call isn't present */
-    HINSTANCE hKernel32;
-    static DWORD(CALLBACK *_GetMaximumProcessorCount)(WORD) = NULL;
-    Py_BEGIN_ALLOW_THREADS
-    hKernel32 = GetModuleHandleW(L"KERNEL32");
-    *(FARPROC*)&_GetMaximumProcessorCount = GetProcAddress(hKernel32,
-        "GetMaximumProcessorCount");
-    Py_END_ALLOW_THREADS
-    if (_GetMaximumProcessorCount != NULL) {
-        ncpu = _GetMaximumProcessorCount(ALL_PROCESSOR_GROUPS);
-    }
-    else {
-        SYSTEM_INFO sysinfo;
-        GetSystemInfo(&sysinfo);
-        ncpu = sysinfo.dwNumberOfProcessors;
-    }
+    DWORD WINAPI GetActiveProcessorCount(WORD group);
```

Comment:
The other option would be to go ahead and set the minimum to Windows 7 in pyconfig.h and backport it to 3.8; as far as I know there are no other things that we want 8+ for yet.

Reply:
```suggestion
    /* Declare prototype here to avoid pulling in all of the Win7 APIs in 3.8 */
    DWORD WINAPI GetActiveProcessorCount(WORD group);
```